ansible/tasks/initdb.yml

```diff
@@ -21,21 +21,51 @@
     user: "{{ db_username }}"
     password: "{{ db_password }}"
     force_basic_auth: yes
-  when: result.status == 404
+  when: dbexists is defined and dbexists.status == 404
```

Comment:
Are those additional guards necessary? Encountered problems with the old versions of just asking for the status code?